launcher/ui/pages/modplatform/ResourcePage.cpp

```diff
@@ -254,10 +254,7 @@ void ResourcePage::updateSelectionButton()
 
     m_ui->resourceSelectionButton->setEnabled(true);
     if (auto current_pack = getCurrentPack(); current_pack) {
-        if (current_pack->versionsLoaded && current_pack->versions.empty()) {
-            m_ui->resourceSelectionButton->setEnabled(false);
-            qWarning() << tr("No version available for the selected pack");
-        } else if (!current_pack->isVersionSelected(m_selected_version_index))
+        if (!current_pack->isVersionSelected(m_selected_version_index))
```

Comment:
I think you included this by mistake

Reply:
Se that is why I said no review.
No, I did not revert the crash fix by mistake.
I need a way to crash the application to test whether I get the stack trace.
But still, something I did wrong as on Windows it doesn't yet work :)